kong/db/schema/entities/upstreams.lua

```diff
@@ -73,6 +73,14 @@ local check_verify_certificate = Schema.define {
   required = true,
 }
 
+local check_headers = Schema.define {
+  type = "array",
```

Comment:
I saw the `unique=true` in array, so was just thinking should this rather be a `set` then?  Is the order a requirement?